bucket4j-spring-boot-starter-context/pom.xml

```diff
@@ -41,7 +41,15 @@
 			<artifactId>spring-boot-configuration-processor</artifactId>
 			<optional>true</optional>
 		</dependency>
-	</dependencies>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-annotations</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.tomcat.embed</groupId>
+            <artifactId>tomcat-embed-core</artifactId>
```

Comment:
What is the reason for this library. Not everyone wants to use a the Tomcat.

Reply:
I believe this was needed for making the predicates work, but i'll check tomorrow to be sure and will get back to you.

Reply:
Looked at it again. It was added for the predicate validations. This however shouldn't have been tomcat, but jakarta.servlet with scope provided, good catch. It is now replaced.